frontend/.gitignore

```diff
@@ -1,3 +1,5 @@
+# WARNING: PLEASE MAKE SURE TO KEEP THIS FILE IN SYNC WITH `.dockerignore`
```

Comment:
this warning is very important!! Otherwise RDEV might not work, because `. dockerignore` might be ignoring things we no longer ignore in `.gitignore`. 

In our case, I updated `.gitignore` to stop ignoring `public/` folder, because Next.js uses it to build the app, but Gatsby used to require us to ignore that folder. But I didn't update `.dockerignore` accordingly, which caused RDEV deploy to not work partially